packages/adapter/addon/index.ts

```diff
@@ -470,6 +470,8 @@ export default class Adapter extends EmberObject implements MinimumAdapterInterf
     return RSVPPromise.resolve();
   }
 
+  _coalesceFindRequests = true;
```

Comment:
We could use `#` private fields here but that will throw an error with non native classes